docs/wiki/installation/cli-flags.md

```diff
@@ -155,6 +160,26 @@ See the **tls**/[remote](../deployment/remote.md) plugin documentation. An enrol
 
 See the **tls**/[remote](../deployment/remote.md) plugin documentation. This is a number of seconds before checking for buffered logs. Results are sent to the TLS endpoint in intervals, not on demand (unless the period=0).
 
+`--distributed_enabled=false`
```

Comment:
Nit, "disable_distributed=false". Most other things use this format. If the near future sees distributed as default enabled but "needs extra configured" then folks will take an explicit disable action. So this sets the stage for a default enabled when we flip the default=true.